README.md

```diff
@@ -271,6 +272,39 @@ logger2 = {"@loggers": "spacy.LookupLogger.v1", "substring": "pytorch"}
 | `cuda_mem_pool`   | `str` | One of the memory pool values specified in the PyTorch docs: `all`, `large_pool`, `small_pool` (default: `all`).                                                |
 | `cuda_mem_metric` | `str` | One of the memory metric values specified in the PyTorch docs: `current`, `peak`, `allocated`, `freed`. To log all metrics, use `all` instead (default: `all`). |
 
+## CuPyLogger
+
+### Installation
+
+This logger requires `cupy` to be installed:
+
+```bash
+pip install cupy
+```
+
+### Usage
+
+Similar to `PyTorchLogger`, `spacy.CuPyLogger.v1` is intended to be used with [ChainLogger](#chainlogger).
+It queries statistics from the CuPy backend and stores them in the info dictionary passed to it. Downstream
+loggers can thereafter lookup the statistics and log them to their preferred framework.
+
+The following CuPy statistics are currently supported:
+
+- [CUDA memory pool statistics](https://docs.cupy.dev/en/stable/user_guide/memory.html)
+
+### Example config
+
+```ini
+[training.logger]
+@loggers = "spacy.ChainLogger.v1"
+logger1 = {"@loggers": "spacy.CuPyLogger.v1", "prefix": "cupy"}
+logger2 = {"@loggers": "spacy.LookupLogger.v1", "patterns": ["cupy"]}
```

Comment:
The `LookupLogger` is exchangeable and processes the data collected by the `CupyLogger`. Correct?

Reply:
Maybe add a comment indicating the the `LookupLogger` can be swapped for any other (equivalent) logger?

Reply:
Done.